recipes/phytop/src/small_tools.py

```
@@ -0,0 +1,358 @@
+			j += 1
+		i += 1
+	wb.save(xls_file)
+
+def combine_tabs_2xls(table_files, xls_file = None, sheets = None):
+	# init
+	if not xls_file:
+		xls_file = table_file[0] + '.xls'
+	if not sheets:
+		sheets = table_file
+
+	wb = xlwt.Workbook()
```

Ruff

Undefined name `xlwt`

(F821)